frontend/src/utils/validateFormField.ts

```diff
@@ -0,0 +1,49 @@
+const validateFormField = (
+  field: string,
+  value: string,
+  password: string = "default"
+) => {
+  let error = "";
+
+  if (field === "name") {
+    if (!value.trim()) {
+      error = "Name is required";
+    } else if (value.trim().length < 3) {
+      error = "Name must be longer than 3 characters";
+    }
+  }
+
+  if (field === "email") {
+    if (!value.trim()) {
+      error = "Email is required";
+    }
+  }
+
+  if (field === "username") {
+    if (!value.trim()) {
+      error = "Username is required";
+    } else if (value.trim().length < 6) {
+      error = "Username must be longer than 6 characters";
+    }
+  }
+
+  if (field === "password") {
+    if (!value.trim()) {
+      error = "Password is required";
+    } else if (value.trim().length < 6) {
+      error = "Username must be longer than 6 characters";
+    }
+  }
+
+  if (field === "confirmPassword") {
+    if (!value.trim()) {
+      error = "Confirm password is required";
+    } else if (value !== password) {
```

Comment:
Do you think this could be better? 🤔  We always receive `password` when `field` is `confirmPassword`. Then in `validateForm` you don't need to check for the case `key == 'confirmPassword'`
```suggestion
    } else if (!!password && value !== password) {
```